results-processor/processor.py

```diff
@@ -123,15 +135,31 @@ def _download_gcs(self, gcs):
 
     def _download_http(self, url):
         assert url.startswith('http://') or url.startswith('https://')
-        _log.debug("Downloading %s", url)
+        _log.debug('Downloading %s', url)
+        extra_headers = None
+        if urlsplit(url).hostname == GITHUB_API_HOSTNAME:
+            extra_headers = {
+                'Authorization': 'Bearer ' + self._github_token,
+                'X-GitHub-Api-Version': GITHUB_API_VERSION,
```

Comment:
```suggestion
                'X-GitHub-Api-Version': self.GITHUB_API_VERSION,
```

This should fix the lint error.